src/Repository/Model/Album.php

```diff
@@ -950,14 +950,14 @@ public function update(array $data)
             }
 
             if (empty($data['album_artist_name']) && !empty($album_artist) && $album_artist != $this->album_artist) {
-                self::update_field('album_artist', $album_artist, $this->id);
```

Comment:
revert this and we're done. it's trying to put a string album name in for artist id